### PR TITLE
SNR move to file-based catalog

### PR DESCRIPTION
**Pull request by clobrano**
https://issues.redhat.com/browse/ECOPROJECT-1008

### CODE REVIEW

**Comment by clobrano** [comment]
/pj-rehearse pull-ci-medik8s-self-node-remediation-main-4.13-openshift-e2e

**Comment by clobrano** [comment]
/pj-rehearse pull-ci-medik8s-self-node-remediation-main-4.15-openshift-e2e

File: ci-operator/config/medik8s/self-node-remediation/medik8s-self-node-remediation-main__4.12.yaml [hunk 1]
```diff
@@ -53,24 +43,41 @@ tests:
...
     test:
-    - as: e2e
-      commands: export SKIP_LOG_VERIFICATION="true" SKIP_OOST_REMEDIATION_VERIFICATION="true"
-        && make e2e-test
-      from: src-oc
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace "$INSTALL_NAMESPACE"
+        operator-sdk run bundle -n "$INSTALL_NAMESPACE" quay.io/medik8s/self-node-remediation-operator-bundle:latest
```
Comment (slintes) [inline ci-operator/config/medik8s/self-node-remediation/medik8s-self-node-remediation-main__4.12.yaml:54]: this line needs to be removed, it's installing SNR from quay, but we want to install the version built by CI in the line below

**Review by slintes — CHANGES_REQUESTED** [review]

File: ci-operator/config/medik8s/self-node-remediation/medik8s-self-node-remediation-main__4.12.yaml (hunk 1 above)
Comment (clobrano) [reply ci-operator/config/medik8s/self-node-remediation/medik8s-self-node-remediation-main__4.12.yaml:54]: 🤦 thank you

**Comment by slintes** [comment]
/pj-rehearse pull-ci-medik8s-self-node-remediation-main-4.14-openshift-e2e

**Comment by slintes** [comment]
/pj-rehearse pull-ci-medik8s-self-node-remediation-main-4.14-openshift-e2e

**Pull request merged**